mm2src/docker_tests.rs

```diff
@@ -1966,49 +1977,46 @@ mod docker_tests {
             "params": {
                 "base": "MYCOIN",
                 "rel": "MYCOIN1",
-                "swap_method": "sell",
+                "swap_method": "setprice",
```

Comment:
What is the reason to put setprice instead of sell here and vice versa below?

Reply:
same comment as below!